Library/Formula/pacapt.rb

```diff
@@ -0,0 +1,15 @@
+class Pacapt < Formula
+  homepage "https://github.com/icy/pacapt"
+  url "https://github.com/icy/pacapt.git",
+    :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4",
+    :tag => "v2.0.2"
+
+  def install
+    bin.mkpath
+    system "make", "install", "BINDIR=#{bin}"
+  end
+
+  test do
+    system "#{bin}/pacapt -Q '.'"
```

Comment:
Fixed, but not tested. My mac broke, it can not boot :(

On 03/31/2015 04:10 PM, Mike McQuaid wrote:
>
> In Library/Formula/pacapt.rb 
> <https://github.com/Homebrew/homebrew/pull/38188#discussion_r27481895>:
>
> > @@ -0,0 +1,15 @@
> > +class Pacapt < Formula
> > +  homepage "https://github.com/icy/pacapt"
> > +  url "https://github.com/icy/pacapt.git",
> > +    :revision => "70cae6c4022696decb6ac1f807a00049e589d9d4",
> > +    :tag => "v2.0.2"
> > +
> > +  def install
> > +    bin.mkpath
> > +    system "make", "install", "BINDIR=#{bin}"
> > +  end
> > +
> > +  test do
> > +    system "#{bin}/pacapt -Q '.'"
>
> |system "#{bin}/pacapt", "-Q", "'.'"| or |system "#{bin}/pacapt", 
> "-Q", "."|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Homebrew/homebrew/pull/38188/files#r27481895>.
>